vmm/src/serial_manager.rs

```diff
@@ -11,10 +11,15 @@ use devices::legacy::Pl011;
 use devices::legacy::Serial;
 use libc::EFD_NONBLOCK;
 use serial_buffer::SerialBuffer;
+use std::ffi::OsString;
 use std::fs::File;
 use std::io::Read;
+use std::net::Shutdown;
+use std::os::fd::IntoRawFd;
```

Comment:
This is causing a build failure on 1.64 - need to bump MSRV?

Reply:
Following https://github.com/rust-lang/rust/blob/bb90f810703af79f6d4006d455bbb6782838854b/library/std/src/os/fd/raw.rs#L114

Seems like this feature is available since 1.4.0. Below is an excerpt the link:

```
#[rustc_allowed_through_unstable_modules]
#[stable(feature = "into_raw_os", since = "1.4.0")]
pub trait IntoRawFd {
```

Investigating this further.